### DISCUSSION TITLE
Birdseye restream with usar and password

### DISCUSSION THREAD

**Discussion by MiAutomations** [blakeblackshear/frigate#14222] [Ask A Question] [answered]
Hello 

I have set the Birdseye restream using this:

birdseye:
  restream: true
  
Is there any way/option to configure a user and a password to access to this stream? 

Thank you

**Comment by NickM-27** [comment, marked as answer]
https://docs.frigate.video/configuration/restream#securing-restream-with-authentication

**Reply by MiAutomations** [reply]
> https://docs.frigate.video/configuration/restream#securing-restream-with-authentication

It is just like this? 

birdseye:
  restream: true
  go2rtc:
  rtsp:
    username: "admin"
    password: "pass"

**Reply by NickM-27** [reply]
go2rtc is its own config, not a property of birdseye

**Reply by MiAutomations** [reply]
Ok Thank you
